internal/runbits/hello_example.go

```diff
@@ -5,10 +5,14 @@ import (
 	"github.com/ActiveState/cli/internal/output"
 )
 
+type NoNameProvidedError struct {
+	*locale.LocalizedError
+}
+
 func SayHello(out output.Outputer, name string) error {
 	if name == "" {
 		// Errors that are due to USER input should use `NewInputError` or `WrapInputError`
-		return locale.NewInputError("hello_err_no_name", "No name provided.")
+		return &NoNameProvidedError{locale.NewInputError("hello_err_no_name", "No name provided.")}
```

Comment:
It just occurred to me that we should not be localizing any errors in the example runner or example runbits (except for the user facing error itself).

Because our ideal is that once all runners speak "User facing error", we no longer have localized errors beyond the user facing ones.

Reply:
I've made the change in this PR though this has required me to introduce a centralized errors file. We will likely need to iterate on this or make the change in a different story/PR.